docs/operator-manual/profiling.md

```diff
@@ -0,0 +1,38 @@
+# Profiling with `pprof`
+
+[`pprof`](https://go.dev/blog/pprof) is a powerful tool for profiling Go applications. It is part of the Go standard library and provides rich insights into the performance characteristics of Go programs, such as CPU usage, memory allocation, and contention.
+
+## Basic Usage
+
+Enable profiling endpoints by setting the environment variable `ARGO_PPROF` with your preferred port. For instance, `ARGO_PPROF=8888` will start profiling endpoints on port `8888`.
+
+`pprof` has two modes: interactive and non-interactive. Non-interactive mode generates profiling data for future analysis. Interactive mode launches a web server to visualize the profiling data in real-time.
+
+!! Note "Replace URLs"
```

Comment:
```suggestion
!!! Note
```

woops, I had a typo here in my previous suggestion, my bad 😅 It's supposed to be _triple_ exclamation.
Also I forgot it doesn't require a title, I think it's cleaner without, since it's only a short sentence